homework-submission/todo-app/src/stores/TasksStore.js

```diff
@@ -0,0 +1,42 @@
+import { observable, action } from "mobx";
+import Data from "../data/todos.json";
+import uuid from 'uuid';
+
+class TasksStore {
+    @observable todos = Data;
+
+    @action
+    handleCheck = id => {
+        const newTodos = this.todos.map(todo => {
```

Comment:
I don't see any use of `.map` here more than the iteration part, while we are looping through all the elements and mutating by there value, we are mutating them directly and not by reference.

with that using `.forEach` is a bit more reasonable.